.github/workflows/build_test_notebooks.yml

```diff
@@ -7,40 +8,45 @@ on:
     branches:
       - tox_update
   workflow_dispatch:
+
 jobs:
   build-notebooks:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout stingray repo
         uses: actions/checkout@v4
         with:
           repository: StingraySoftware/stingray
           path: ./stingray
+
       - name: Checkout notebooks repo
         uses: actions/checkout@v4
         with:
           repository: AnonymousCodes911/notebooks  
           ref: tox_update  
           path: ./notebooks
+
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
           python-version: '3.10'
+
       - name: Install pandoc and dependencies
         run: |
           sudo apt-get update
           sudo apt-get install -y pandoc fonts-freefont-ttf
-          pip install sphinx sphinx-astropy nbsphinx tox 
+          pip install sphinx sphinx-astropy tox
+
       - name: Install Stingray
-        run: pip install -e git+https://github.com/StingraySoftware/stingray.git#egg=stingray
+        run: pip install -e ./stingray
+
       - name: Update submodules
         run: |
+          cd ./stingray
```

Comment:
What you are updating here is the notebook submodule version already in stingray. You should add an instruction to checkout the exact version of the notebooks of the PR

Reply:
fixed in latest commit